### PR TITLE
Feature/gal 313

### PR DESCRIPTION
**Pull request by Abdul-NHS**
## Description
Ticket: https://nhsd-jira.digital.nhs.uk/browse/GAL-313

The function created appends the moderator column to unique lsoa csv, which is then uploaded to the unique lsoa table. These moderator values are derived from bowel screening.

- [x] Refactoring (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [x] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: scripts/test_data/updateForecastUptakeMod/updateForecastUptakeMod.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    console.log("CSV file saved successfully");
+  });
+};
+
+// read in both csvs
+const lsoaUniqueArray = await processData(lsoaData);
+const arrWithMod = await processData(LsoaModerators);
+
+//cut moderator to 3dp
+const newModArr = fixDecimal(arrWithMod);
```
Comment (zainm97) [inline scripts/test_data/updateForecastUptakeMod/updateForecastUptakeMod.js:56]: can you can just do this in the processData function? If you pass the processData function another function to call and fix the decimal there. Would avoid nested loop

**Review by zainm97 — APPROVED** [review]
Nice one

**Review by Amar-NHS — COMMENTED** [review]
Looks good. Nice work Abdul. as discussed you will delete the table and rebuild again due to limit exceed for update

**Review by Amar-NHS — APPROVED** [review]
Looks good. Nice work Abdul. as discussed you will delete the table and rebuild again due to limit exceed for update

**Pull request merged**